components/chart/overview.md

```diff
@@ -12,11 +12,9 @@ position: 0
 
```

Comment:
Each Overview page should have a titled section for all features that are explained on a separate dedicated page. If the features are too many, you can [group them in one section or enumerate them in a list](https://github.com/telerik/blazor/wiki/Docs-Overview-Page-Template). See the [Grid Overview](https://docs.telerik.com/blazor-ui/components/grid/overview) and sections "Data Operations", "Column Features" and "More Grid Features".